alpine/tools-deps/Dockerfile

```diff
@@ -0,0 +1,19 @@
+#
+# NOTE: THIS DOCKERFILE IS GENERATED VIA "update.sh"
+#
+# PLEASE DO NOT EDIT IT DIRECTLY.
+#
+
+FROM openjdk:8-alpine
+MAINTAINER Kirill Chernyshov <delaguardo@gmail.com>
+
+ENV CLOJURE_VERSION=1.9.0.358
+
+WORKDIR /tmp
+
+RUN wget https://download.clojure.org/install/linux-install-$CLOJURE_VERSION.sh
+    && chmod +x linux-install-$CLOJURE_VERSION.sh
+    && ./linux-install-$CLOJURE_VERSION.sh
```

Comment:
This script does not seem to run in alpine because of missing 'curl' command

Reply:
Thanks, I forgot to push latest changes 🤦‍♂️